website/docs/d/account_settings.html.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: "linode"
+page_title: "Linode: linode_account_settings"
+sidebar_current: "docs-linode-datasource-account-settings"
+description: |-
+Provides information about Linode account settings.
+---
+
+# linode\_account\_settings
+
+Provides information about Linode account settings.
+
+## Example Usage
+
+The following example shows how one might use this data source to access information about Linode account settings.
+
+```hcl
+data "linode_account_settings" "example" {}
+```
+
+## Attributes Reference
+
+Each Linode LKE Version will be stored in the `versions` attribute and will export the following attributes:
```

Comment:
I think you can drop/edit this line 👍

Reply:
Good catch! I missed that when I copy-pasted :)